apps/game_client/test/lambda_game_backend_web/controllers/page_controller_test.exs

```diff
@@ -1,8 +1,8 @@
 defmodule GameClientWeb.PageControllerTest do
```

Comment:
Could this file be directly removed? Or do we need to keep it for some reason?

Reply:
Yeah definitely